samples/SampleApp/appsettings.Production.json

```diff
@@ -9,6 +9,11 @@
           "Password": "testPassword"
         }
       }
+    },
+    "Certificates": {
+      "Default": {
```

Comment:
Use a different node with it's own defined semantics. E.g. dockercert or appdatacert.